src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/DigitalGreenCertificateAccess.swift

```diff
@@ -29,29 +31,26 @@ public struct DigitalGreenCertificateAccess {
         }
     }
 
-    public func extractCBORWebTokenHeader(from cborData: CBORData) -> Result<CBORWebTokenHeader, CertificateDecodingError> {
-        return extractHeader(from: cborData)
-    }
-
     public func extractDigitalGreenCertificate(from base45: Base45) -> Result<DigitalGreenCertificate, CertificateDecodingError> {
         let cborDataResult = extractCBOR(from: base45)
 
         switch cborDataResult {
         case let .success(cborData):
-            return extractDigitalGreenCertificate(from: cborData)
+            return extractCertificate(from: cborData)
         case let .failure(error):
             return .failure(error)
         }
     }
 
-    public func extractDigitalGreenCertificate(from cborData: CBORData) -> Result<DigitalGreenCertificate, CertificateDecodingError> {
-        return extractCertificate(from: cborData)
-    }
-
     // MARK: - Internal
 
     func extractCBOR(from base45: Base45) -> Result<CBORData, CertificateDecodingError> {
-        guard let zipData = try? base45.fromBase45() else {
+        guard base45.hasPrefix(hcPrefix) else {
+            return .failure(.HC_COSE_TAG_INVALID)
```

Comment:
```suggestion
            return .failure(.HC_PREFIX_INVALID)
```

(I guess it needs to be "HC" instead of "VC" like it is in the tech spec)

Reply:
Yes I think it needs to bo HC. Because from the perspective of the package it can happen to both VC and PC. Added HC_PREFIX_INVALID. And also use it now 🤦‍♂️